core/src/main/java/org/kiwix/kiwixmobile/core/main/MainMenu.kt

```diff
@@ -171,7 +160,7 @@ class MainMenu(
 
   fun tryExpandSearch(zimFileReader: ZimFileReader?) {
     if (search.isVisible) {
-      zimFileReader?.let(::navigateToSearch)
+      zimFileReader?.apply { navigateToSearch() }
```

Comment:
`apply` isn't so idiomatic, in this case I think you can just use an `if` or replace the apply with a `let`